README.md

```diff
@@ -36,10 +36,10 @@ routines have been substantially altered.</b>
 
 To access, use the following in your analysis code:
 ```
-from csu_radartools import (csu_fhc, csu_liquid_ice_mass, csu_blended_rain, csu_dsd, 
-                            csu_kdp, csu_misc)
+from csu_radartools import (csu_fhc, csu_liquid_ice_mass, csu_blended_rain, csu_dsd,
+                            csu_kdp, csu_misc, fundamentals, common)
```

Comment:
Feel that `common` should be an internal import, not really an externally advertised module. The `fundamentals` should be the public face of what you are doing.